cmd/bucket-handlers-listobjects.go

```diff
@@ -70,8 +70,18 @@ func (api objectAPIHandlers) ListObjectsV2Handler(w http.ResponseWriter, r *http
 		return
 	}
 
+	urlValues := r.URL.Query()
```

Comment:
The logic seems right, but I think it belongs to  `func getListObjectsV2Args(...)`.
Could you move it inside `func getListObjectsV2Args(...)`?

Reply:
Hi @ebozduman  , Yes but will it be ugly sending an errorCode in `getListObjectsV2Args` ?

Reply:
I don't think so. IMHO, it belongs there. 
We get the args, input validate them, error out if required, otherwise we pass the args to the caller  if they qualify. 
That's how I see it, unless there is a different style we follow.
@krishnasrinivas, what do you think?

Reply:
> Hi @ebozduman , Yes but will it be ugly sending an errorCode in getListObjectsV2Args ?

It is not ugly to return errorCode from getListObjectsV2Args(), it is just a value like `error interface`. We return errorCode from a lot of functions at S3 Layer.
ex:
```
func extractSignedHeaders(signedHeaders []string, r *http.Request) (http.Header, APIErrorCode)
```

Doing the validation outside or inside the getListObjectsV2Args() is not a big deal, but it is a better place to do this validation inside getListObjectsV2Args() so that it can take care of all the validation "details".

Reply:
@krishnasrinivas  Okay . Got it !
@ebozduman made the change !